deploy/compose/compose.env

```diff
@@ -6,6 +6,8 @@ export MODEL_DIRECTORY="/home/nvidia/llama2_13b_chat_hf_v1/"
 # Fill this out if you dont have a GPU. Leave this empty if you have a local GPU
 export NVIDIA_API_KEY="nvapi-*"
 
+export AI_PLAYGROUND_API_KEY="nvapi-*"
```

Comment:
This should not be needed. We use ChatNvidia from [langchain-nvidia-ai-endpoints](https://python.langchain.com/docs/integrations/chat/nvidia_ai_endpoints) which uses `NVIDIA_API_KEY`.

Reply:
Hi @nv-pranjald, thank you for your feedback! I have incorporated your comments into the relevant files. Please help review again!
Regarding the parameter 'AI_PLAYGROUND_API_KEY', it is utilized in the 'docker-compose-pgvector.yaml' file. I once encountered a situation where the 'AI_PLAYGROUND_API_KEY' definition was missing in the source environment file. I want to confirm that if it is not assigned a value, it will automatically be given a default value. Therefore, I have removed it.